src/App.jsx

```diff
@@ -1,22 +1,65 @@
-import { useState } from 'react';
+import { collection, query, where, getDocs, terminate } from 'firebase/firestore';
+import { useState, useEffect } from 'react';
 import './App.css';
 import Home from './pages/Home';
+import db from '../src/utils/firebase';
 
 function App() {
   const [count, setCount] = useState(0);
+  const [entries, setEntries] = useState([]);
+  const [terms, setTerms] = useState({});
+  const [users, setUsers] = useState([]);
 
+  useEffect(() => {
```

Comment:
I heard it's better to use useQuery for fetching data than useEffect

Reply:
I can use it but I'd like to get the prototype working with the live data first.

Reply:
or that could be a good issue for someone once we finish.

Reply:
ok. when the page is rerendered, i can see multiple YRL. we can fix this on another pr

Reply:
I will fix the re-rendering. I'm opening a new merge tonight. I want to fix and add a few things.